style.css

```
@@ -0,0 +1,49 @@
+html,
+body {
+  margin: 0px;
+  height: 100%;
+  overflow: hidden;
+}
+body {
+  background: radial-gradient(circle, #f6fffc, #bdebdf);
+}
+h1 {
+  font-style: italic;
+  padding-left: 30px;
+}
+a {
+  text-decoration: none;
+  color: black;
+  padding: 5px 15px;
+  box-shadow: 5px 0px black;
```

Când ai valoarea unei proprietăți 0, o poți lăsa fără unitate de măsură.
```suggestion
  box-shadow: 5px 0 black;
```